### ISSUE TITLE
Execution Reverted with RollupCreator.createRollup on Ethereum Sepolia

### ISSUE THREAD

**Issue by pc-quiknode** [OffchainLabs/arbitrum-orbit-sdk#215]
Hello There,

I am seeing a pretty high failure rate of the createRollup function that happens randomly.

Example TX: https://sepolia.etherscan.io/tx/0x01f69b5a029c94d6cc187b9bb59e9ed6bb84bce037ccd4b240ed3db49bf0b5c4

Inputs:
```
{
  "deployParams": {
    "config": {
      "confirmPeriodBlocks": 150,
      "extraChallengeTimeBlocks": 0,
      "stakeToken": "0x0000000000000000000000000000000000000000",
      "baseStake": "100000000000000000",
      "wasmModuleRoot": "0x184884e1eb9fefdc158f6c8ac912bb183bf3cf83f0090317e0bc4ac5860baa39",
      "owner": "0x84f52e68fb1f2fed32b5e1ac6bdee6c3bf12bed8",
      "loserStakeEscrow": "0x0000000000000000000000000000000000000000",
      "chainId": "991515267",
      "chainConfig": "{\"homesteadBlock\":0,\"daoForkBlock\":null,\"daoForkSupport\":true,\"eip150Block\":0,\"eip150Hash\":\"0x0000000000000000000000000000000000000000000000000000000000000000\",\"eip155Block\":0,\"eip158Block\":0,\"byzantiumBlock\":0,\"constantinopleBlock\":0,\"petersburgBlock\":0,\"istanbulBlock\":0,\"muirGlacierBlock\":0,\"berlinBlock\":0,\"londonBlock\":0,\"clique\":{\"period\":0,\"epoch\":0},\"arbitrum\":{\"EnableArbOS\":true,\"AllowDebugPrecompiles\":false,\"DataAvailabilityCommittee\":false,\"InitialArbOSVersion\":32,\"GenesisBlockNum\":0,\"MaxCodeSize\":24576,\"MaxInitCodeSize\":49152,\"InitialChainOwner\":\"0x84f52e68FB1f2Fed32b5e1Ac6bDEe6C3bf12bED8\"},\"chainId\":991515267}",
      "genesisBlockNum": 0,
      "sequencerInboxMaxTimeVariation": {
        "delayBlocks": "28800",
        "futureBlocks": "300",
        "delaySeconds": "345600",
        "futureSeconds": "3600"
      }
    },
    "validators": [
      "0x83fe1185a2c8f9d3394f6a6be3373f85f93edc50"
    ],
    "maxDataSize": "117964",
    "nativeToken": "0x0000000000000000000000000000000000000000",
    "deployFactoriesToL2": true,
    "maxFeePerGasForRetryables": "100000000",
    "batchPosters": [
      "0x7dea51885bfb4d1b187d26dd13a5831f2187a00a"
    ],
    "batchPosterManager": "0x0000000000000000000000000000000000000000"
  }
}
```

**Comment by spsjvc** [comment]
Hey @pc-quiknode, which version are you using? We had some issues related to this that were fixed in [v0.20.1](https://github.com/OffchainLabs/arbitrum-orbit-sdk/releases/tag/v0.20.1).

**Comment by pc-quiknode** [comment]
> Hey @pc-quiknode, which version are you using? We had some issues related to this that were fixed in [v0.20.1](https://github.com/OffchainLabs/arbitrum-orbit-sdk/releases/tag/v0.20.1).

Hello @spsjvc 
I am using [0.21.0](https://github.com/OffchainLabs/arbitrum-orbit-sdk/releases/tag/v0.21.0)

**Comment by spsjvc** [comment]
@pc-quiknode we're gonna do a patch release soon that adds some extra buffer so that this doesn't happen

in the meantime, to unblock yourself, you can override `value` in `createRollupPrepareTransactionRequest` with a value of 0.13 ether which should be plenty to cover for the fees, and the excess will be refunded

**Comment by spsjvc** [comment]
@pc-quiknode we just released [v0.21.1](https://github.com/OffchainLabs/arbitrum-orbit-sdk/releases/tag/v0.21.1) which should fix this. 

Closing this issue now, but don't hesitate to reopen if it keeps happening.
